go/vt/vtgate/evalengine/testcases/inputs.go

```diff
@@ -199,6 +199,40 @@ var inputStrings = []string{
 	// "_ucs2 'AabcÅå'",
 }
 
+var insertStrings = []string{
+	"NULL",
+	"\"\"",
+	"\"a\"",
+	"\"abc\"",
+	"1",
+	"-1",
+	"0123",
+	"0xAACC",
+	"3.1415926",
+	// MySQL has broken behavior for these inputs,
```

Comment:
You can reference https://github.com/mysql/mysql-server/pull/517 here so we know what this is about in the future too.

Reply:
done.